ephios/static/ephios/js/main.js

```diff
@@ -31,6 +31,13 @@ $(document).ready(function () {
             $('#checkall').prop('checked', false);
         }
     });
+
+    $("[data-jslink]").on("click", function (ev) {
```

Comment:
i'm not sure if I like this solution, mainly because context actions don't work on mobile (so I cannot hold the event and select "open in a new tab" and I hate it when web apps do that)